.travis.yml

```diff
@@ -19,9 +19,7 @@ jobs:
       name: "Node JS 12"
```

Comment:
can you doctor the above to remove Node 8, Node 10, and Node 12 - making the tests run on Node JS 16 as well?

Reply:
theoretically it looks like you could leave 12 maybe, but 8 is not passing on either travis job

Reply:
bumped it and it seems to have gotten further.